eth/gasprice/feehistory.go

```diff
@@ -88,6 +93,16 @@ func (oracle *Oracle) processBlock(bf *blockFees, percentiles []float64) {
 		bf.results.nextBaseFee = new(big.Int)
 	}
 	bf.results.gasUsedRatio = float64(bf.header.GasUsed) / float64(bf.header.GasLimit)
+	bf.results.blobGasUsedRatio = 0.0
+	bf.results.blobBaseFee = new(big.Int)
+	if bf.header != nil && chainconfig.IsCancun(bf.header.Number, bf.header.Time) {
+		if blobGasUsed := bf.header.BlobGasUsed; blobGasUsed != nil {
+			bf.results.blobGasUsedRatio = float64(*blobGasUsed) / params.MaxBlobGasPerBlock
+		}
+		if excessBlobGas := bf.header.ExcessBlobGas; excessBlobGas != nil {
+			bf.results.blobBaseFee = eip4844.CalcBlobFee(*excessBlobGas)
```

Comment:
Ok, so I think I need to look at the _previous_ block for `blobBaseFee`, then add a `nextBlobBaseFee` that should contain the value calculated from the current excess blob gas.

But currently I don't have access to the previous header in this function.

Reply:
Added bf.parentHeader in cc1649a.

Reply:
Oh I see, `excess_blob_gas` is actually the already computed from the parent header so ya you're right. 

> The value of excess_blob_gas can be calculated using the parent header.